lib/saml/user.rb

```diff
@@ -30,7 +34,31 @@ def authn_context
     private
 
     def serializable_attributes
-      @decorated.send(:serializable_attributes) + %i(authn_context last_signed_in)
+      @decorated.serializable_attributes + %i(authn_context last_signed_in)
+    end
+
+    # see warnings
+    def log_warnings_to_sentry!
+      warning_context = {
+        authn_context: authn_context || 'nil ~= idme',
+        warnings: warnings.join(', '),
+        loa: @decorated.try(:loa)
+      }
+      log_message_to_sentry('Issues in SAML Response LOA', :warn, warning_context)
```

Comment:
Shouldn't this have an `unless warnings.empty?` or equivalent?

Reply:
yes, good catch.